schemas/logs/aws/tests/cloud_trail_digest_tests.yml

```diff
@@ -60,10 +62,9 @@ result: |
         ],
         "p_log_type": "AWS.CloudTrailDigest"
   	}
-results: []
```

Comment:
Why is this removed?

Reply:
I set it to `omitempty`

Reply:
We usually want `result`  not `results`.